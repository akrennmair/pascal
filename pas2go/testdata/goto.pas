program test;

label 10;

var i : integer;

begin
    i := 0;
    10:
    i := i + 1;
    writeln('i = ', i);
    if i < 10 then
        goto 10
end.