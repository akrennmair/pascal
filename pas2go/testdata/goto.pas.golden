package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	var (
		i int
	)
	_ = i

	i = 0
L10:
	i = i + 1
	system.Writeln("i = ", i)
	if i < 10 {
		goto L10
	}
}
